zap/src/main/java/org/zaproxy/zap/extension/pscan/ExtensionPassiveScan.java

```diff
@@ -294,7 +295,9 @@ protected PassiveScannerList getPassiveScannerList() {
             listTest.addAll(ExtensionFactory.getAddOnLoader().getPassiveScanRules());
 
             for (PluginPassiveScanner scanner : listTest) {
-                addPluginPassiveScannerImpl(scanner);
+                if (!addPluginPassiveScannerImpl(scanner)) {
+                    LOGGER.error("Failed to install pscanrule: {}", scanner.getName());
+                }
```

Comment:
We should either skip adding or logging when a `RegexAutoTagScanner`, it's logging an error always:
```
[ZAP-BootstrapGUI] ERROR org.zaproxy.zap.extension.pscan.ExtensionPassiveScan - Failed to install pscanrule: null
```
since `addPluginPassiveScannerImpl` returns false for `RegexAutoTagScanner`.